toolbox/CMakeLists.txt

```diff
@@ -275,6 +273,10 @@ else()
     target_link_libraries(WBToolbox ${LINKED_LIBRARIES} ${Matlab_LIBRARIES})
 endif()
 
+# Link with ClockServer library
+add_subdirectory(autogenerated/)
+target_link_libraries(WBToolbox PUBLIC ClockServer)
```

Comment:
Move these two lines next to the Synchronizer block, so in the future you can `if` the whole block

Reply:
It is not possible moving this portion of code before the target `WBToolbox` is created